matcher_test.go

```diff
@@ -0,0 +1,84 @@
+//go:build linux
+
+package goss
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/goss-org/goss/util"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	update = flag.Bool("update", false, "update the golden files of this test")
+)
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+	os.Exit(m.Run())
+}
+
+func TestMatchers(t *testing.T) {
+	files, err := filepath.Glob(filepath.Join("testdata", "out_matching_*"))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for _, outFile := range files {
+		outFile := outFile
+		parts := strings.Split(outFile, ".")
+		tfName := fmt.Sprintf("%s.yaml", strings.TrimPrefix(parts[0], "testdata/out_"))
```

Comment:
What's tf in this context? Please expand or comment?

Reply:
"test file" renamed to specFile